development/common/integrations/authentication/originMiddleware.ts

```
@@ -1,4 +1,9 @@
 
 import { OriginMiddleware } from '@jitar-plugins/http';
 
-export default new OriginMiddleware();
+const options = {
+    sameSite: process.env.ORIGIN_COOKIE_SAME_SITE || 'Strict',
+    secure: process.env.ORIGIN_COOKIE_SECURE === 'true'
+}
+
```

ESLint

[error] Missing semicolon.

(semi)